hca/util/__init__.py

```diff
@@ -115,9 +115,19 @@ def special_cli_command(self, required_argument, optional_argument=""):
 
 
 class RetryPolicy(retry.Retry):
-    def __init__(self, retry_after_status_codes={301}, *args, **kwargs):
+    def __init__(self,
+                 retry_after_status_codes={301},
+                 read=10,
+                 status=10,
+                 backoff_factor=0.1,
```

Comment:
Since we're not sure what we're fixing, perhaps we should adopt a conservative philosophy. I suggest leaving `RetryPolicy` unchanged except for `status_forcelist`.